### PR TITLE
Prompt for HTTPS/SSL connection types

### PR DESCRIPTION
**Pull request by jshcrowthe**
This is a PR to address #248.

I have set the default connection type to be `HTTPS` but that can be changed if desired/needed.

### CODE REVIEW

**Comment by samccone** [comment]
LGTM :+1: thanks @jshcrowthe

File: gh/index.js [hunk 1]
```diff
@@ -35,13 +35,19 @@ module.exports = yeoman.generators.Base.extend({
         name: 'branch',
         message: 'What branch would you like to deploy from?',
         default: 'master'
-      }
+      },
+      {
+        name: 'connectionType',
+        message: 'Use HTTPS or SSL?',
```
Comment (robdodson) [inline gh/index.js]: Would you mind changing this to:

'Use HTTPS or SSL to connect to GitHub?'

Just to help first time users

Comment (jshcrowthe) [inline gh/index.js]: No problem let me get that updated!

Comment (samccone) [inline gh/index.js]: :dancer:

**Comment by jshcrowthe** [comment]
I have updated the prompt (per @robdodson), would you like me to squash commits and re-push the branch?

**Comment by samccone** [comment]
+1 to a squash if you would not mind @jshcrowthe

**Comment by robdodson** [comment]
yup squash/rebase would be appreciated. thank you!

**Comment by jshcrowthe** [comment]
@robdodson, @samccone Squashed and Rebased! Interestingly enough Travis build failed for iojs. I reran the tests locally and was unable to reproduce the error:

screen shot 2015-12-18 at 10 00 25 am

Not sure what happened there.

**Comment by robdodson** [comment]
oh it's probably fine. let me rerun the tests.

**Comment by robdodson** [comment]
All good now. Thanks @jshcrowthe !

**Comment by jshcrowthe** [comment]
Awesome thanks so much @robdodson  and @samccone. #255 can also be closed.

Also thanks for your patience on this, took far too long to do the 15 line PR.

**Pull request merged**